formats/ome-tiff/code.txt

```diff
@@ -28,11 +28,21 @@ the following steps are required:
 
 #. Read in the 8-byte header.
 #. Determine if the file is a valid TIFF, and if so, whether its byte
-   order is big endian or little endian. The first pair of bytes must
-   equal "II" (little endian, "Intel") or "MM" (big endian, "Motorola").
-   The next pair must equal 42 with the proper endianness.
-#. Determine the byte offset into the file of the first IFD. This
-   information is stored in bytes 4-7, with the proper endianness.
+   order is big endian or little endian. Bytes 0-1 must equal "II"
+   (little endian, "Intel") or "MM" (big endian, "Motorola"), which is
+   0x4D4D in the correct endian format.
+#. Check TIFF version.  Bytes 2-3 must equal 42 (0x2A) or 43 (0x2B)
+   with the proper endianness, which are the TIFF specification or the
+   newer BigTIFF specification, respectively.
+#. If the TIFF version is 0x2A, offsets are 4 bytes (32-bit).  If the
+   TIFF version is 0x2B, bytes 4-5 are the size of offsets in bytes
+   (should be 0x0008 for 8-byte 64-bit offsets, but could be
+   different), and bytes 6-7 are padding (should be 0x0000).
+#. Determine the byte offset into the file of the first IFD. For TIFF
+   version 0x2A, this information is stored in bytes 4-7, with the
+   proper endianness.  For TIFF version 0x2B, this information is
```

Comment:
"with the proper endianness" here raises questions for me of, for instance, how about that 0x0008 above, is that also of the proper endianness?